examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -7073,7 +7073,7 @@ endpoint 1 {
     ram      attribute coupleColorTempToLevelMinMireds;
     persist  attribute startUpColorTemperatureMireds;
     ram      attribute featureMap default = 0x1F;
-    ram      attribute clusterRevision default = 5;
+    ram      attribute clusterRevision default = 6;
```

Comment:
This should not be done to land before 1.2 tag, since otherwise someone going to SVE could claim Rev 6 support. Please do not change until we are opened up for 1.3